toolkit/tools/roast/formats/qcow.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package formats
+
+import (
+	"fmt"
+
+	"microsoft.com/pkggen/internal/shell"
+)
+
+const (
+	// QcowType represents the qcow2 virtual drive format
+	QcowType = "qcow2"
+)
+
+// Qcow implements Converter interface to convert a RAW image into a qcow2 file
+type Qcow struct {
+}
+
+// Convert converts the image in the qcow2 format
+func (v *Qcow) Convert(input, output string, isInputFile bool) (err error) {
+	if !isInputFile {
+		return fmt.Errorf("qcow2 conversion requires a RAW file as an input")
+	}
+
+	err = shell.ExecuteLive(false, "qemu-img", "convert", "-O", "qcow2", input, output)
```

Comment:
Should we be explicit about our intent: [like we have done in `vhd.go`]
```suggestion
	const (
		squashErrors = false
		outputFormat = "qcow2"
	)
	err = shell.ExecuteLive(squashErrors, "qemu-img", "convert", "-O", outputFormat, input, output)

```

Reply:
Thanks Falak, I made the changes